apps/civicsignalblog/src/payload/lib/data/common/applications.js

```diff
@@ -1,7 +1,7 @@
-export const MAIN = "main";
+export const MAIN = "tools";
 export const EXPLORER = "explorer";
-export const TOPIC_MAPPER = "topic-mapper";
-export const SOURCE_MAPPER = "source-manager";
+export const TOPIC_MAPPER = "topics";
+export const SOURCE_MAPPER = "sources";
```

Comment:
I like the original names i.e. `topic-mapper`, `source-mapper` compared to `topics` and `sources`. Any reason why we're changing them?

Reply:
I had to rename them  to match what is in the [webtools.](https://github.com/CodeForAfrica/web-tools/blob/main/src/config.js)

That way, I can easily/consistently make API request for content tied with an app name using the `AppName-collection` naming scheme like how I did it [here](https://github.com/CodeForAfrica/web-tools/blob/2e4983a36b9937d2dfab859e2d94f637c5d2d92c/src/components/tools/media/Homepage.js)

Reply:
Good find... all good.